lib/actions/addLinkToProjectAction.js

```diff
@@ -12,21 +13,21 @@ class AddLinkToProjectAction {
     }
     
     let linkParams = { 
-      label: parsedUserInput[0],
-      link: parsedUserInput[1]
+      label: parsedUserInput[1],
+      link: parsedUserInput[2]
     }
-      
     return linkParams
   }
 
-  async execute(params){ 
+  async execute(params, currentProject){ 
     let paramsObj = this.parseParams(params)
-    // project.links[label] = link;     
-    // store data -- > await db.store() if it is async
-    // throw new Error('uh oh.. something went wrong');
-    return new AddLinkToProjectResponse(paramsObj.label, paramsObj.link)  
+    const project = await currentProject;
```

Comment:
You don't need to `await` an instance of a `Project`. Instead you would `await` an async method call that returns a promise. `currentProject` is not a promise here.